packages/create-astro/src/actions/template.ts

```diff
@@ -23,7 +23,7 @@ export async function template(
 			choices: [
 				{ value: 'basics', label: 'Include sample files', hint: '(recommended)' },
```

Comment:
```suggestion
				{ value: 'basics', label: 'A basic, minimal starter', hint: '(recommended)' },
```
Again, don't feel strongly, but I feel like "include sample files" was probably in part to distinguish it from "empty" project, and now this might feel like "But, I don't want your files, I want a new fresh project from scratch!"

So, maybe highlighting somehow that this *is* honestly a minimal starter could be helpful?